big_tests/tests/gdpr_SUITE.erl

```diff
@@ -225,8 +255,64 @@ retrieve_roster(Config) ->
                 Alice, Config, "roster", ExpectedHeader, ExpectedItems)
         end).
 
-retrieve_mam(_Config) ->
-    ok.
+retrieve_mam_pm(Config) ->
+    F = fun(Alice, Bob) ->
+            Msg1 = <<"some simple pm message">>,
+            Msg2 = <<"another simple pm message">>,
+            Msg3 = <<"third simple pm message">>,
+            escalus:send(Alice, escalus_stanza:chat_to(Bob, Msg1)),
+            escalus:send(Bob, escalus_stanza:chat_to(Alice, Msg2)),
+            escalus:send(Alice, escalus_stanza:chat_to(Bob, Msg3)),
+            mam_helper:wait_for_archive_size(Alice, 3),
+
+            ExpectedHeader = ["id", "message"],
+            ExpectedItems = [
+                                #{"message" => [{contains, Msg1}]},
+                                #{"message" => [{contains, Msg3}]}
+                            ],
+
+            BackendModule = case proplists:get_value(mam_backend, Config) of
+                                rdbms -> mod_mam_rdbms_arch;
+                                riak -> mod_mam_riak_timed_arch_yz;
+                                cassandra -> [mod_mam_cassandra_arch, mod_mam_cassandra_arch_params];
+                                elasticsearch -> mod_mam_elasticsearch_arch
+                            end,
+            maybe_stop_and_unload_module(mod_muc, BackendModule, Config),
```

Comment:
Didnt you man `mod_mam` ?

Reply:
right, that's obvious mistake. thanks for catching it